telegram/inline/inputcontactmessagecontent.py

```diff
@@ -48,3 +48,5 @@ def __init__(self, phone_number, first_name, last_name=None, vcard=None, **kwarg
         # Optionals
         self.last_name = last_name
         self.vcard = vcard
+
+        self._id_attrs = (self.phone_number, self.first_name)
```

Comment:
One *could* argue, that `last_name` should be added. That reduces the risk of evaluating to people with the same (first) name as equal.

Reply:
Then again, for `Contact`, we only use phone number. That seems cleaner to me, as the same phone number shouldn't appear twice.

Reply:
Agreed. Phone number should be enough to uniquely identify someone, first name on top is nice to have, but definitely enough